diff --git a/demo/atis_joint_model/data_processor.py b/demo/atis_joint_model/data_processor.py
index 81cda0f..0cc9e2d 100644
--- a/demo/atis_joint_model/data_processor.py
+++ b/demo/atis_joint_model/data_processor.py
@@ -5,7 +5,7 @@ import os
 import random

 import click
-
+from pytext.main import ClickLocaleCommand

 VALIDATION_SPLIT = 0.25
 SAMPLE_PRINT_COUNT = 5
@@ -165,7 +165,7 @@ def print_sample(file_name):
             print(line)


-@click.command()
+@click.command(cls=ClickLocaleCommand)
 @click.option("-d", "--download-folder", required=True, type=str)
 @click.option("-o", "--output-directory", required=True, type=str)
 @click.option("-v", "--verbose", default=False, type=bool)
diff --git a/pytext/main.py b/pytext/main.py
index 4ae77aa..3896d34 100644
--- a/pytext/main.py
+++ b/pytext/main.py
@@ -152,8 +152,35 @@ def gen_config_impl(task_name, options):
             raise Exception(f"Unknown option: {opt}")
     return config_to_json(PyTextConfig, root)
 
-
-@click.group()
+def set_locale():
+    import os
+    import platform
+    platform = platform.platform()
+    if 'redhat' in platform:
+        os.environ['LC_ALL'] = 'en_US.UTF8'
+    elif 'debian' in platform :
+        os.environ['LC_ALL'] = 'C.UTF-8'
+        os.environ['C.UTF-8'] = 'C.UTF-8'
+
+class ClickLocaleGroup(click.core.Group):
+    def __init__(self, name=None, commands=None, **attrs):
+        set_locale()
+        click.core.Group.__init__(self, name, commands, **attrs)
+
+
+class ClickLocaleCommand(click.core.Command):
+    def __init__(self, name, context_settings=None, callback=None,
+                 params=None, help=None, epilog=None, short_help=None,
+                 options_metavar='[OPTIONS]', add_help_option=True,
+                 hidden=False, deprecated=False):
+        set_locale()
+        click.core.Command.__init__(self, name, context_settings, callback,
+                                    params, help, epilog, short_help,
+                                    options_metavar, add_help_option,
+                                    hidden, deprecated)
+
+
+@click.group(cls=ClickLocaleGroup)
 @click.option(
     "--include", multiple=True, help="directory containing custom python classes"
 )
