diff --git a/pytext/main.py b/pytext/main.py
index 02ad59c..ead0258 100644
--- a/pytext/main.py
+++ b/pytext/main.py
@@ -349,12 +349,24 @@ def train(context):
 @click.option("--output-onnx-path", help="where to save the exported onnx model")
 @click.pass_context
 def export(context, model, output_path, output_onnx_path):
-    """Convert a pytext model snapshot to a caffe2 model."""
+    """Convert a pytext model snapshot to a caffe2 model.
+
+    If model is provided, path for exported models will then be loaded
+    from the supplied parameters rather than any passed config file.
+    Otherwise, a config file will be loaded.
+    """
+
     if not model:
         config = context.obj.load_config()
         model = config.save_snapshot_path
         output_path = config.export_caffe2_path
         output_onnx_path = config.export_onnx_path
+
+    if output_path is None:
+        raise Exception(
+            "if --model is set --output-path must be set"
+        )
+
     print(
         f"Exporting {model} to caffe2 file: {output_path} and onnx file: {output_onnx_path}"
     )
