diff --git a/pytext/metrics/__init__.py b/pytext/metrics/__init__.py
index ec68b51..ea3bef7 100644
--- a/pytext/metrics/__init__.py
+++ b/pytext/metrics/__init__.py
@@ -242,7 +242,7 @@ class ClassificationMetrics(NamedTuple):
                 "avg_pr": "Average precision",
                 "roc_auc": "ROC AUC",
             }
-            print(ascii_table(soft_scores, columns))
+            print(ascii_table(soft_scores, columns).encode('utf-8'))
             all_thresholds = set(
                 itertools.chain.from_iterable(
                     metrics.recall_at_precision
@@ -267,7 +267,7 @@ class ClassificationMetrics(NamedTuple):
                         **{str(t): f"R@P {t}" for t in all_thresholds},
                     ),
                     alignments={"label": "<"},
-                )
+                ).encode('utf-8')
             )
         if self.mcc:
             print(f"\nMatthews correlation coefficient: {self.mcc :.3f}")
