From 6234026ff24e2cb68ff582096d8e9ff1069dc294 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Tue, 2 Jan 2018 15:40:19 +0000
Subject: [PATCH 03/18] Win32: Do not link -static-libgcc

---
 src/gnuwin32/fixed/etc/Makeconf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gnuwin32/fixed/etc/Makeconf b/src/gnuwin32/fixed/etc/Makeconf
index 4f024e43e6..6ff08f7500 100644
--- a/src/gnuwin32/fixed/etc/Makeconf
+++ b/src/gnuwin32/fixed/etc/Makeconf
@@ -11,8 +11,8 @@ else
   DEBUGFLAG=
 endif
 
-DLLFLAGS+= -static-libgcc
-LINKFLAGS+= -static-libgcc
+DLLFLAGS+=
+LINKFLAGS+=
 
 ## Things which are substituted by fixed/Makefile (and also -O3 -> -O2)
 WIN = 32
-- 
2.20.1

