From 5da573fff6a0b0b778a82865631ffa2028b87447 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Tue, 2 Jan 2018 15:40:19 +0000
Subject: [PATCH 03/10] Win32: Do not link -static-libgcc

---
 src/gnuwin32/fixed/etc/Makeconf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gnuwin32/fixed/etc/Makeconf b/src/gnuwin32/fixed/etc/Makeconf
index 79bc1e41e2..4a26ca82b6 100644
--- a/src/gnuwin32/fixed/etc/Makeconf
+++ b/src/gnuwin32/fixed/etc/Makeconf
@@ -11,8 +11,8 @@ else
   DEBUGFLAG=
 endif
 
-DLLFLAGS+= -static-libgcc
-LINKFLAGS+= -static-libgcc
+DLLFLAGS+=
+LINKFLAGS+=
 
 ## Things which are substituted by fixed/Makefile (and also -O3 -> -O2)
 WIN = 32
-- 
2.14.3 (Apple Git-98)

