--- r-base.orig/src/library/tcltk/R/unix/zzz.R	2018-02-07 04:14:38.000000000 -0800
+++ r-base/src/library/tcltk/R/unix/zzz.R	2018-02-07 04:17:03.000000000 -0800
@@ -49,7 +49,7 @@
             if(length(ind)) {
                 this <- sub(" .*", "", sub("^\t", "", out[ind]))
                 ##  message("tcltk DLL is linked to ", shQuote(this))
-                if(!file.exists(this))
+                if(!file.exists(this) && !grepl("@rpath",this))
                     stop("Tcl/Tk libraries are missing: install the Tcl/Tk component from the R installer",
                          domain = NA)
             }
