diff --git a/src/ray/thirdparty/hiredis/Makefile b/src/ray/thirdparty/hiredis/Makefile
index cff2a84..2447654 100644
--- a/src/ray/thirdparty/hiredis/Makefile
+++ b/src/ray/thirdparty/hiredis/Makefile
@@ -36,7 +36,7 @@ endef
 export REDIS_TEST_CONFIG

 # Fallback to gcc when $CC is not in $PATH.
-CC:=$(shell sh -c 'type $(CC) >/dev/null 2>/dev/null && echo $(CC) || echo gcc')
+CC?=$(shell sh -c 'type $(CC) >/dev/null 2>/dev/null && echo $(CC) || echo gcc')
 CXX:=$(shell sh -c 'type $(CXX) >/dev/null 2>/dev/null && echo $(CXX) || echo g++')
 OPTIMIZATION?=-O3
 WARNINGS=-Wall -W -Wstrict-prototypes -Wwrite-strings
