diff --git a/BUILD.bazel b/BUILD.bazel
index ed0af09..437ce1c 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -827,7 +827,7 @@ genrule(
     cmd = """
         set -x &&
         curl -sL \"https://github.com/antirez/redis/archive/5.0.3.tar.gz\" | tar xz --strip-components=1 -C . &&
-        make &&
+        make CC=$$CONDACC &&
         mv ./src/redis-server $(location redis-server) &&
         chmod +x $(location redis-server) &&
         mv ./src/redis-cli $(location redis-cli) &&
diff --git a/build.sh b/build.sh
index 1e149c2..11bcf24 100755
--- a/build.sh
+++ b/build.sh
@@ -112,7 +112,7 @@ if [ "$RAY_BUILD_JAVA" == "YES" ]; then
 fi

 if [ "$RAY_BUILD_PYTHON" == "YES" ]; then
-  $BAZEL_EXECUTABLE build //:ray_pkg --verbose_failures
+  $BAZEL_EXECUTABLE build //:ray_pkg --action_env=CONDACC=$CC --verbose_failures
 fi

 popd
