diff --git a/h5py/tests/test_file.py b/h5py/tests/test_file.py
index 2a09fd3..9a0137a 100644
--- a/h5py/tests/test_file.py
+++ b/h5py/tests/test_file.py
@@ -255,17 +255,6 @@ class TestDrivers(TestCase):
         self.assertTrue(fid)
         fid.close()
 
-    @ut.skipUnless(mpi, "Parallel HDF5 is required for MPIO driver test")
-    def test_mpio(self):
-        """ MPIO driver and options """
-        from mpi4py import MPI
-
-        comm=MPI.COMM_WORLD
-        fname = self.mktemp_mpi(comm)
-        with File(fname, 'w', driver='mpio', comm=comm) as f:
-            self.assertTrue(f)
-            self.assertEqual(f.driver, 'mpio')
-
     @ut.skipUnless(mpi, "Parallel HDF5 required")
     @ut.skipIf(h5py.version.hdf5_version_tuple < (1, 8, 9),
                "mpio atomic file operations were added in HDF5 1.8.9+")
